common/src/main/scala/org/apache/celeborn/common/util/FunctionConverter.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.celeborn.common.util
 
+import scala.language.implicitConversions
```

Comment:
why do we need this import?

Reply:
> FunctionConverter.scala:26: implicit conversion method scalaFunctionToJava should be enabled
by making the implicit value scala.language.implicitConversions visible.
This can be achieved by adding the import clause 'import scala.language.implicitConversions'
or by setting the compiler option -language:implicitConversions.
See the Scaladoc for value scala.language.implicitConversions for a discussion
why the feature should be explicitly enabled.